main.go

```diff
@@ -211,13 +215,10 @@ func main() {
 
 	mux.Handle(apiPath+"/v1/", http.StripPrefix(apiPath+"/v1", av1))
 
-	server := &http.Server{
-		Addr:    *listenAddress,
-		Handler: mux,
-	}
+	server := &http.Server{Handler: mux}
 
 	go shutdownServerOnQuit(server, quitCh, logger)
-	err = web.Serve(l, server, *webConfig, logger)
+	err := web.ServeMultiple(listeners, server, webConfig, logger)
```

Comment:
Rather than use `ServeMultiple`, we could switch this to `web.ListenAndServe()`. Then we can drop the whole section at line 155.

Reply:
I see. I changed it, and it seems to work.